Outlook.md

```diff
@@ -0,0 +1,25 @@
+Based on the *DataHub version* of d:swarm, the following features are planned to support the improvement of data quality and structure:
```

Comment:
I think we should also link to smaller achievements here, i.e., not only the "greater" visions, but things like "direct RDF export" etc.